cmake/FindBlazeIterative.cmake

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2018 Patrick Diehl
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+find_package(PkgConfig)
```

Comment:
This will not work on non-Linux platforms that don't have pkgconfig available.

Reply:
Good comment, I will add support for Windows as well.

Reply:
https://github.com/STEllAR-GROUP/hpx/blob/master/cmake/FindOrangeFS.cmake#L7
https://github.com/STEllAR-GROUP/hpx/blob/master/cmake/FindJemalloc.cmake#L10

Ok, for HPX we do it similar and I assume that the QUIET option should help to get it run on Windows or?